packages/call-to-action/index.js

```diff
@@ -1,7 +1,25 @@
 import Button from '@hashicorp/react-button'
+import classNames from 'classnames'
+import variantCentered from './styles/variant-centered.module.css'
+import variantCompact from './styles/variant-compact.module.css'
+import variantLinks from './styles/variant-links.module.css'
 
-function CallToAction(props) {
-  const { heading, content, links, variant, product, theme } = props
+const stylesDict = {
+  centered: variantCentered,
+  compact: variantCompact,
+  links: variantLinks,
+}
+
+function CallToAction({
+  heading,
+  content,
+  links,
+  variant = 'centered',
+  product,
+  theme = 'light',
+  className,
+}) {
+  const s = stylesDict[variant]
```

Comment:
Took a slightly different approach here from other components, as this component uses "variants" with fully separated styles that have aligned `classNames` within each `.module.css` file. Open to other ideas if this feels too unorthodox.